feature-libs/my-account/organization/src/assets/translations/en/user.i18n.ts

```diff
@@ -0,0 +1,88 @@
+export const user = {
+  header: 'Users',
+  disabled: '(disabled)',
+  uid: 'Code',
+  name: 'Name',
+  firstName: 'Fist name',
+  lastName: 'Last name',
+  email: 'Email',
+  orgUnit: 'Parent Unit',
+  roles: 'Roles',
+  title: 'Title',
+  assignAprover: 'Add user to the approver list for this user',
```

Comment:
Should be 'Add user to the approver list for this unit', as in https://cxwiki.sap.com/pages/viewpage.action?spaceKey=spar&title=B2B+My+Company-+Users